modules/default/calendar/calendar.js

```diff
@@ -368,7 +369,7 @@ Module.register("calendar", {
 						} else {
 							timeWrapper.innerHTML = this.capFirst(
 								moment(event.startDate, "x").calendar(null, {
-									sameDay: "[" + this.translate("TODAY") + "]",
+									sameDay: this.config.showTimeToday ? "LT" : "[" + this.translate("TODAY") + "]",
```

Comment:
So if this is set, it displays "LT"? How does that string show up on the MM?

Reply:
It passes LT as the format string to Moment (compare the default value of `dateEndFormat` or other explicit uses [here](https://github.com/MichMich/MagicMirror/blob/c3339b47bb6ccfc8bdf927f9582d40217332a5ad/modules/default/calendar/calendar.js#L308) and [here](https://github.com/MichMich/MagicMirror/blob/c3339b47bb6ccfc8bdf927f9582d40217332a5ad/modules/default/calendar/calendar.js#L312).  LT renders time according to the current locale, e.g. "2:53 PM", "14:53", etc.